repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -186,27 +203,38 @@ class Interpreter(prompt0: Ref[String],
     }
   }
 
-  def processLine(code: String, stmts: Seq[String], fileName: String): Res[Evaluated] = for{
-    _ <- Catching { case ex =>
-      Res.Exception(ex, "Something unexpected went wrong =(")
-    }
+  def processLine(code: String,
+                  stmts: Seq[String],
+                  fileName: String,
+                  preprocess: Preprocessor = Preprocessor(compiler.parse)
```

Comment:
Why is `preprocess` a default argument? Is someone going to pass in a custom version? If not it should be a normal `val` defined in the method body